storage_service/storage_service/settings/components/auth.py

```diff
@@ -0,0 +1,42 @@
+from os import environ
```

Comment:
This is much cleaner now, and hopefully starts to pave a way for a more significant refactor of the original file. Thank you! 

If you feel so inclined it would be great if you could add a module docstring - I think it just helps adds the belt and braces to making storage service settings become more descriptive. Otherwise, I really appreciate this.